requirements.txt

```diff
@@ -1,11 +1,14 @@
 annotated-types==0.7.0
```

Comment:
Adding `anyio` without context is concerning.  Please provide details in the PR description about why this dependency is needed and what functionality it provides within the application.  Specifically, how does it relate to the addition of the OpenAI model and `AIService`?  Lacking this context makes this change hard to review and assess for potential security or performance implications.